onnxruntime.sh

```diff
@@ -0,0 +1,58 @@
+package: onnxruntime
+version: "1.7.2"
+tag: c073c88ef
+source: https://github.com/saganatt/onnxruntime.git
+requires:
+  - protobuf
+  - re2
+build_requires:
+  - CMake
+---
+#!/bin/bash -e
+
+pushd $SOURCEDIR
+  git submodule update --init -- cmake/external/date
```

Comment:
There is no alternative than doing git submodule init?

Reply:
hm, what only comes to my mind is that the users would need to install these dependencies themselves.